### ISSUE TITLE
How I can add colum of actions in table, without edit the /node_modules/ng2-table/com..

### ISSUE THREAD

**Issue by leonardovff** [valor-software/ng2-table#409]
I need to modify the markup of the table, but I can't to do this without modifying the component in node_modules.

**Comment by webcat12345** [comment]
Good question. You can add string(HTML string) to the table. then I will work as you expected.

```
    person.actionEdit = '<a class="action-btn edit"><i class="fa fa-pencil-square-o" aria-hidden="true"></i></a>';
    person.actionDelete = '<a class="action-btn remove"><i class="fa fa-trash-o" aria-hidden="true"></i></a>';
```

```
public onCellClick(data: any, delete_confirm: any): any {
    if (data.column === 'actionEdit'){
```

**Comment by leonardovff** [comment]
This is very bad :(

**Comment by seiyria** [comment]
Unfortunately doing that, while it would work, is extremely non-ideal and makes this basically unusable for people who need actions.

**Comment by AndrewJHart** [comment]
Yeah - i was wondering the same thing actually. I need to do a bulk-select with checkboxes (or select individual rows) and the only way i've found to do that is by attaching markup directly to my model (view-model?) and it seems unclean. Anyone else have any thoughts on this?

**Comment by danilomcb** [comment]
Didn't like the option of @webcat12345 either. I made a fork and implemented so it's possible to add more than one action button at the last column. It's a simple implementation, that following the same pattern from the ng2-table team.

Basically, I created a object that represents the action buttons that should be displayed at the table. The action may be identified by it's type.
```javascript
export const TableActions:Array<any> = [
 {
   'title': '',
   'type': 'edit',
   'tooltip': 'Edit',
   'classBtn': '',
   'classIcon': 'fa fa-pencil',
 },
 {
   'title': '',
   'type': 'delete',
   'tooltip': 'Delete',
   'classBtn': '',
   'classIcon': 'fa fa-trash',
 }
];
```

Already send pull request to the ng2-table team (#467). Waiting the answer. :slightly_smiling_face:

**Comment by johnlee** [comment]
@danilomcb  I had a similar issue where i wanted some checkboxes in my table. I'm looking at your suggestion and wondering if that would work for checkboxes. I think it will not because you still have a click event handler in that last column. (?) Were there any other suggestions on how to do checkboxes?

**Comment by kesemdavid** [comment]
@danilomcb any update on that? looking for that feature

**Comment by danilomcb** [comment]
Exactly @johnlee . In your case, you would have to make the change to work with the checkboxes.

**Comment by danilomcb** [comment]
@kesemdavid I created a project fork and am using it. I think it was not updated with my improvement.

**Comment by isasiluis28** [comment]
@danilomcb any idea if they will update the project with your commits?
